bindings/java/src/org/sleuthkit/datamodel/AccountFileInstance.java

```diff
@@ -1,7 +1,7 @@
 /*
  * Sleuth Kit Data Model
  *
- * Copyright 2011-2017 Basis Technology Corp.
+ * Copyright 2017 Basis Technology Corp.
```

Comment:
I thought we had been using 2011-current year for all copyright headers?

Reply:
For new files, it doesn't make sense to have dates that are before the file existed.

Reply:
What has happened here is that I tried to standardize on the policy of using a copyright for the software for every file. Brian disagrees with this policy. It may, in fact, be incorrect. Unfortunately, we have lots of "my style" copyright notices in place now.